cosmos/runtime/app.go

```diff
@@ -310,7 +311,17 @@ func NewPolarisApp( //nolint: funlen // from sdk.
 
 	// Polaris Configuration
 	cfg := provider.DefaultConfig()
-	cfg.NodeConfig.DataDir = DefaultNodeHome + "/data"
+	homePath, ok := appOpts.Get(flags.FlagHome).(string)
```

Comment:
This isn't the spot for this. 


I would probably do it in eth/provider and then we should pass the filepath down thru.